### PR TITLE
Enhancement: description field added to targets

### PR DESCRIPTION
**Pull request by ivanfelfo**

### CODE REVIEW

File: db/migrate/20210514171510_add_description_field_to_targets.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class AddDescriptionFieldToTargets < ActiveRecord::Migration[6.1]
+  def change
+    change_table :targets, bulk: true do |t|
+      t.string :description
```
Comment (martinjaimem) [inline db/migrate/20210514171510_add_description_field_to_targets.rb:4]: what about enforcing it is not null?

Comment (martinjaimem) [reply db/migrate/20210514171510_add_description_field_to_targets.rb:4]: and maybe adding a presence validation in the model too 🤔

Comment (ivanfelfo) [reply db/migrate/20210514171510_add_description_field_to_targets.rb:4]: please, don't atomize me :)

Comment (juanmferreira93) [reply db/migrate/20210514171510_add_description_field_to_targets.rb:4]: agree, both things

File: spec/requests/targets/targets_create_spec.rb [hunk 2]
```diff
@@ -2,7 +2,8 @@
   let(:user) { create(:user) }
   let(:topic) { create(:topic) }
   let(:params) do
-    { target: { title: 'Burgers', latitude: 1, longitude: 1, radius: 2, topic_id: topic.id } }
+    { target: { title: 'Burgers', latitude: 1, longitude: 1, radius: 2, topic_id: topic.id,
```
Comment (antoalconti) [inline spec/requests/targets/targets_create_spec.rb:5]: Here maybe you can use ` attributes_for(:target,   topic_id: topic.id)`, if you already have target factory

File: db/migrate/20210514171510_add_description_field_to_targets.rb (hunk 1 above)
Comment (ivanfelfo) [reply db/migrate/20210514171510_add_description_field_to_targets.rb:4]: we discussed with Martin and came to the conclusion that :description should not be presence validated

**Review by juanmferreira93 — APPROVED** [review]

File: spec/requests/targets/targets_create_spec.rb (hunk 2 above)
Comment (ivanfelfo) [reply spec/requests/targets/targets_create_spec.rb]: 👍

File: db/migrate/20210514171510_add_description_field_to_targets.rb (hunk 1 above)
Comment (martinjaimem) [reply db/migrate/20210514171510_add_description_field_to_targets.rb:4]: Yeah, but there are a lot of other fields we didn't add these validations in the `db`, let's add them here 😄

Comment (ivanfelfo) [reply db/migrate/20210514171510_add_description_field_to_targets.rb:4]: I added the NOT NULL constraint to title, user_id and topic_id, which I feel makes more sense than applying it to description. What you guys think?

**Review by martinjaimem — APPROVED** [review]

**Pull request merged**